web/Services/TenantService.cs

```diff
@@ -19,15 +19,15 @@ public TenantService(BricksAndHeartsDbContext dbContext, IPostcodeService postco
         _postcodeService = postcodeService;
     }
 
-    public async Task<List<TenantDbModel>?> SortTenantsByLocation(string postalCode, int page, int perPage)
+    public async Task<IEnumerable<TenantDbModel>> SortTenantsByLocation(string postalCode)
     {
         var postcode = _postcodeService.FormatPostcode(postalCode);
         var postcodeList = new List<string> { postcode };
         await _postcodeService.AddPostcodesToDatabaseIfAbsent(postcodeList);
         var targetLocation = _dbContext.Postcodes.SingleOrDefault(p => p.Postcode == postcode);
         if (targetLocation?.Lat == null || targetLocation.Lon == null)
         {
-            return null;
+            return null!;

```

Comment:
Could this not return an empty list instead?

Reply:
Yes, I considered that before but making an empty IQueryable/IEnumerable (which would have been required before) was more difficult to figure out - now that everything will return Lists again, I've made it return as an empty list.

Reply:
So an empty list is an IEnumerable. A List implement IList, which extends ICollection, which extends IEnumerable, so a List implements the interface IEnumerable :)